dashboard/app/controllers/learning_goal_evaluations_controller.rb

```diff
@@ -25,6 +25,45 @@ def update
     end
   end
 
+  def get_evaluation
+    learning_goal_evaluation = LearningGoalEvaluation.find_by(
+      user_id: learning_goal_evaluation_params[:user_id],
+      teacher_id: current_user.id,
+      learning_goal_id: learning_goal_evaluation_params[:learning_goal_id]
+    )
+
+    if learning_goal_evaluation&.teacher_id == current_user.id
+      render json: learning_goal_evaluation
+    else
+      head :not_found
+    end
+  end
+
+  def get_or_create_evaluation
+    learning_goal_evaluation = LearningGoalEvaluation.find_or_create_by(
+      user_id: learning_goal_evaluation_params[:user_id],
+      teacher_id: current_user.id,
+      learning_goal_id: learning_goal_evaluation_params[:learning_goal_id]
+    )
+    if learning_goal_evaluation.created_at
+      render json: learning_goal_evaluation
+    else
+      learning_goal_evaluation.created_at = DateTime.current
+      learning_goal_evaluation.save!
+      render json: learning_goal_evaluation
+    end
+  end
+
+  def index
+    learning_goal_evaluations = LearningGoalEvaluation.all
+    render json: learning_goal_evaluations
+  end
+
+  def show
+    learning_goal_evaluations = LearningGoalEvaluation.find(params[:id])
+    render json: learning_goal_evaluations
+  end
```

Comment:
sorry if I missed this but where are these routes used?

Reply:
Oh, I was using those for debug. I'll get rid of them.